server/game/cards/07-GR/DeEscalation.js

```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+
+class DeEscalation extends Card {
+    // Play: Destroy each creature. Your opponent archives the top 3
+    // cards of their deck.
+    setupCardAbilities(ability) {
+        this.play({
+            effect: 'destroy all creatures{1}',
+            effectArgs: (context) => [
+                context.player.opponent
+                    ? ' and have ' +
+                      context.player.opponent.name +
+                      ' archive the top 3 cards of their deck'
+                    : ''
+            ],
+            gameAction: [
```

Comment:
this one needs to be sequential. If opponent has a Rad Penny, for example, they must be able to archive it.

Reply:
Good catch!  Fixed, thanks.